gate_code/gate/observe/controller.py

```diff
@@ -0,0 +1,33 @@
+from django.http import JsonResponse
+from time import time
+from random import random
+from os import environ
+
+def start_game(request):
+    environ['PLAY_FLG'] = '1'
+    return JsonResponse({'status':200})
+
+def get_position_list(requset):
+    data = []
+    if environ['PLAY_FLG'] == '1':
+        # On play
+        now = time()
+        for i in range(50):
+            data.append({
+                'timestamp' : int((now + 0.1 * i) * 1000),
+                'location' : {
```

Comment:
In this cord. Snitch will appear discontinuously. I guess that we can find more continuous function.

Reply:
Thanks you for comment. But this is stub api, so I guess  you do not need to consider about it.
Please check the structure of response